tests/contract_testcases/substrate/selector_override.dot

```diff
@@ -1,30 +1,32 @@
-	i [label="function i\ncontract: selector\ntests/contract_testcases/substrate/selector_override.sol:6:2-45\nsignature i()\nvisibility internal\nmutability nonpayable"]
-	p [label="function p\ncontract: selector\ntests/contract_testcases/substrate/selector_override.sol:7:2-44\nsignature p()\nvisibility private\nmutability nonpayable"]
-	diagnostic [label="found contract 'selector'\nlevel Debug\ntests/contract_testcases/substrate/selector_override.sol:1:1-8:2"]
-	diagnostic_10 [label="selector is 2 bytes, 4 bytes expected\nlevel Error\ntests/contract_testcases/substrate/selector_override.sol:2:25-34"]
-	diagnostic_11 [label="overriding selector not permitted on modifier\nlevel Error\ntests/contract_testcases/substrate/selector_override.sol:3:24-29"]
-	diagnostic_12 [label="overriding selector not permitted on receive\nlevel Error\ntests/contract_testcases/substrate/selector_override.sol:4:38-44"]
-	diagnostic_13 [label="overriding selector not permitted on fallback\nlevel Error\ntests/contract_testcases/substrate/selector_override.sol:5:31-39"]
-	diagnostic_14 [label="overriding selector only permitted on 'public' or 'external' function, not 'internal'\nlevel Error\ntests/contract_testcases/substrate/selector_override.sol:6:35-45"]
-	diagnostic_15 [label="overriding selector only permitted on 'public' or 'external' function, not 'private'\nlevel Error\ntests/contract_testcases/substrate/selector_override.sol:7:34-44"]
+	contract [label="contract selector\ntests/contract_testcases/substrate/selector_override.sol:1:1-14:2"]
+	new [label="constructor new\ncontract: selector\ntests/contract_testcases/substrate/selector_override.sol:3:2-16\nsignature new()\nvisibility public\nmutability nonpayable\nselector abcd"]
+	m [label="modifier m\ncontract: selector\ntests/contract_testcases/substrate/selector_override.sol:5:2-15\nmutability nonpayable"]
+	underscore [label="undersore\ntests/contract_testcases/substrate/selector_override.sol:5:16-17"]
+	node_5 [label="receive \ncontract: selector\ntests/contract_testcases/substrate/selector_override.sol:7:2-28\nmutability payable"]
+	node_6 [label="fallback \ncontract: selector\ntests/contract_testcases/substrate/selector_override.sol:9:2-21\nmutability nonpayable"]
+	i [label="function i\ncontract: selector\ntests/contract_testcases/substrate/selector_override.sol:11:2-23\nsignature i()\nvisibility internal\nmutability nonpayable"]
+	p [label="function p\ncontract: selector\ntests/contract_testcases/substrate/selector_override.sol:13:2-22\nsignature p()\nvisibility private\nmutability nonpayable"]
+	diagnostic [label="found contract 'selector'\nlevel Debug\ntests/contract_testcases/substrate/selector_override.sol:1:1-14:2"]
+	diagnostic_11 [label="function 'new' selector 'abcd' must be 4 bytes rather than 2 bytes\nlevel Error\ntests/contract_testcases/substrate/selector_override.sol:2:2-24"]
```

Comment:
What does this message look like when you have an array of integers (not hex)?

Reply:
Good catch. This got lost when re-writing this. Fixed.